python/rikai/types/vision.py

```diff
@@ -93,13 +102,18 @@ def from_pil(img: PILImage, uri: Union[str, Path]) -> Image:
             An PIL Image instance
         uri : str or Path
             The URI to store the image externally.
-        """
+        format : str, optional
+            The image format to save as. See
+            `supported formats <https://pillow.readthedocs.io/en/stable/reference/Image.html#PIL.Image.Image.save>`_ for details.
+        kwargs : dict, optional
+            Optional arguments to pass to `PIL.Image.save <https://pillow.readthedocs.io/en/stable/reference/Image.html#PIL.Image.Image.save>`_.
+        """  # noqa: E501
         parsed = urlparse(normalize_uri(uri))
         if parsed.scheme == "file":
-            img.save(uri)
+            img.save(uri, format=format, **kwargs)
```

Comment:
Just curious, what is the implication that we just keep `format` in `**kwargs`?

Reply:
PIL.Image separates it out so i just did it for consistency. 🤷